lib/Core/Executor.cpp

```diff
@@ -290,10 +290,17 @@ namespace {
 		    clEnumValEnd),
 		  cl::ZeroOrMore);
 
+#if LLVM_VERSION_CODE < LLVM_VERSION(3, 0)
   cl::opt<unsigned int>
   StopAfterNInstructions("stop-after-n-instructions",
                          cl::desc("Stop execution after specified number of instructions (default=0 (off))"),
                          cl::init(0));
+#else
+  cl::opt<unsigned long long>
```

Comment:
@delcypher @MartinNowack I quickly tried right now and there is no available template instantiation for `uint64_t` on LLVM 3.4. Sincerely I'd like to avoid to write a parser since we already have one working for `unsigned long long`.